features/support/save_to_dir_embedding_formatter.rb

```diff
@@ -34,8 +34,20 @@ def initialize(config)
 
     # @note #embed has been replaced by #attach in Cucumber 5.x
     def attach(src, media_type)
-      if media_type != 'text/x.cucumber.log+plain'
-        embed(src, media_type, "test case execution attachment")
+      if media_type == 'text/x.cucumber.log+plain'
+        puts src
+      elsif media_type == 'image/png;base64'
+        filename = Time.now.iso8601 + '-screenshot.png'
+        info "embedding #{filename}"
```

Comment:
With the new changes, it reports error when `info` is called seems it is only accessible in `log.rb`

```
05-20 16:33:49.206        undefined method `info' for #<BushSlicer::SaveToDirEmbeddingFormatter:0x0000000005a81f38> (NoMethodError)
05-20 16:33:49.206        /home/jenkins/ws/workspace/ocp-common/Runner/features/support/save_to_dir_embedding_formatter.rb:41:in `attach'
05-20 16:33:49.207        /home/jenkins/ws/workspace/ocp-common/Runner/lib/log.rb:162:in `embed'
```

Reply:
Use `puts` to replace `info`